diff -crB -N ./build/_deps/unity-src/src/unity_config.h ./config/unity_config.h
*** ./build/_deps/unity-src/src/unity_config.h	1970-01-01 00:00:00.000000000 +0000
--- ./config/unity_config.h	2024-09-01 23:01:18.458973500 +0100
***************
*** 0 ****
--- 1,6 ----
+ #ifndef UNITY_INCLUDE_CONFIG_H
+ #define UNITY_INCLUDE_CONFIG_H
+ 
+ #define UNITY_SUPPORT_64
+ 
+ #endif
\ No newline at end of file